### ISSUE TITLE
No network?

### ISSUE THREAD

**Issue by muhv7** [lincanbin/Android-Carbon-Forum#12]
Sorry, have to bother you again.

So I've done everything so far, changed the API secrets, changed the domain names etc. Made an API subdomain and binded it to the homepage. 

Still getting No network whenever I open the app.

Either Im missing something right now or Im just dumb .

**Comment by lincanbin** [comment]
![tim 20171206172706](https://user-images.githubusercontent.com/5785188/33654324-b2ebb06a-daaa-11e7-9f14-b3547dadea85.png)
You may have missed this.

**Comment by lincanbin** [comment]
When you visit the API domain name, have you looked `403`?

**Comment by muhv7** [comment]
Hey. Tried with https and without. Either way both show me No network in the app. 

403 as in error? The api page itself is currently redirecting fine to the page. (Unless I misunderstood you in the other topic and it should do something else...)

**Comment by lincanbin** [comment]
* Your api domain name is resolved and bound to the current server?

![tim 20171206174302](https://user-images.githubusercontent.com/5785188/33655033-ef35890e-daac-11e7-9677-12af079ab620.png)
* Do you set the api domain name in Dashboard?

![tim 20171206174432](https://user-images.githubusercontent.com/5785188/33655096-23ebfdd6-daad-11e7-8901-79e22b6652ff.png)
If API works, then open it should be like this.

**Comment by muhv7** [comment]
Yup.. I think Im just stupid and missing something. 

Dashboard is set as shown on your picture but at this point I'm just thinking I missed to upload something to the API domain part.

**Comment by lincanbin** [comment]
You do not need to upload anything.

**Comment by muhv7** [comment]
Yeah. That's the issue. I made an sub domain. CNAME'd it to the forum root, set the API path in dashboard and it's still saying Network error

**Comment by lincanbin** [comment]
When you open your own api domain name in the browser, saw what?

**Comment by muhv7** [comment]
Index of ...
I know you said it should be 403 but for whatever reason that does not show up for me..

**Comment by lincanbin** [comment]
![tim 20171208165501](https://user-images.githubusercontent.com/5785188/33758278-8c1a76ee-dc38-11e7-91a2-e9b8a5d418a9.png)
I think you did not bind the api domain name to vhost.

**Comment by pratikraman-dev** [comment]
hello @lincanbin  i have done all above things getting 403 error also still getting no network in android app

**Comment by lincanbin** [comment]
@pratt1305 The 403 status code appears on behalf of the server configuration is correct, check your client configuration.

**Comment by pratikraman-dev** [comment]
Client Configuration Means Android - Side ??

**Comment by lincanbin** [comment]
@pratt1305 https://github.com/lincanbin/Android-Carbon-Forum/blob/master/app/src/main/java/com/lincanbin/carbonforum/config/APIAddress.java#L15

**Comment by pratikraman-dev** [comment]
This Setting Are Done Perfectly 
![image](https://user-images.githubusercontent.com/8825570/38288386-1e91e0a0-37ee-11e8-8105-5d97b352858c.png)

**Comment by lincanbin** [comment]
Does your site have HTTPS support? @pratt1305

**Comment by pratikraman-dev** [comment]
No

**Comment by lincanbin** [comment]
![tim 20180404122849](https://user-images.githubusercontent.com/5785188/38288487-be5de2b8-3803-11e8-88ab-d12545951b71.png)
@pratt1305

**Comment by lincanbin** [comment]
Remove `s` in `https`.

**Comment by pratikraman-dev** [comment]
Still Same issue after removing S from https://

![image](https://user-images.githubusercontent.com/8825570/38289001-a0ccac3c-37f1-11e8-913b-e1668800d4e9.png)

**Comment by lincanbin** [comment]
把日志发上来

**Comment by pratikraman-dev** [comment]
ok

**Comment by lincanbin** [comment]
Send logs up.

**Comment by lincanbin** [comment]
Your api seems to be inaccessible.

**Comment by pratikraman-dev** [comment]
![image](https://user-images.githubusercontent.com/8825570/38289191-b25799ac-37f2-11e8-8590-70a7d36a28ca.png)

**Comment by pratikraman-dev** [comment]
How Can We test those api's are working or not ?? Please point me in some direction so i go through it

**Comment by pratikraman-dev** [comment]
Where is path to api in Carbon PHP server i think its coming from view/api

**Comment by lincanbin** [comment]
403 means working.

**Comment by pratikraman-dev** [comment]
alright !! then exactly where can i be going wrong ??

**Comment by lincanbin** [comment]
From the log, your API returns an empty string to the client.

**Comment by jishnurajm** [comment]
@lincanbin I'm also facing the same issue! How to fix this 
![screenshot from 2018-05-05 17-58-36](https://user-images.githubusercontent.com/27047484/39663287-207f55a6-508e-11e8-9696-da6888504603.png)

**Comment by jish-44** [comment]
@lincanbin i'm also facing the same issue, The android app shows an network error still after done the settings are perfectly. 
![screenshot from 2018-05-05 18-03-42](https://user-images.githubusercontent.com/27047484/39663315-aacb1696-508e-11e8-8eee-bb24fc1c1632.png)
![screenshot from 2018-05-05 18-03-30](https://user-images.githubusercontent.com/27047484/39663316-ab282746-508e-11e8-9a2f-2b9076fdd227.png)

**Comment by lincanbin** [comment]
@jishnurajm Domain `api.statushub.net` is not bound to your virtual host. 
Bind it in cpanel.

**Comment by mercado** [comment]
How do you do this in the cpanel? I tried: public_html/view/api,  but it gave the same error of the second print of @jishnurajm

**Comment by BlessedEmperorconcept** [comment]
hello please am having the same issue
Please help
